Makefile.release

```diff
@@ -0,0 +1,68 @@
+SHELL := /bin/bash
```

Comment:
adding a new rule to the existing makefile might be better.

this is ok as a quick solution, we can make it nicer later

Reply:
I'm following the idea of @patrickjahns where two makefiles exist: one for dev and one for release. To be discussed ....